skyxplore-ui-test/src/test/java/com/mentoring/test/EquipmentTest.java

```diff
@@ -80,15 +81,36 @@ public void createAndSetUpAnEquipment() throws SQLException {
         softAssertion.assertThat(factoryPage.getQuantityInQueue().equals(DEFAULT_QUANTITY))
                 .as("Quantity in queue is not correct")
                 .isTrue();
-        softAssertion.assertThat(factoryPage.isQueueProcessStarted())
-                .as("Queue process is not started.")
+        softAssertion.assertThat(factoryPage.isQueueProcessBarVisible())
+                .as("Queue process bar is not visible.")
                 .isTrue();
 
         log.info("Updating queue process end time to finish the production.");
         userId = DbSelects.getUserIdByEmailAddress(database, user.getEmail());
         characterId = DbSelects.getCharacterIdByUserId(database, userId);
         factoryId = DbSelects.getFactoryIdByCharacterId(database, characterId);
-        //TODO: DbUpdates
+        DbUpdates.setProductionEndTimeByFactoryId(database, factoryId, "0");
+
+        driver.navigate().refresh();
+
+        try {
+            Thread.sleep(10000);
+        } catch (InterruptedException e) {
+            log.error("Thread sleep failed.", e);
+        }
```

Comment:
You should not do this. There must be always something that you can wait for.

Reply:
Yes, of course, sorry. I know. It is only a work version. (It was late night and couldn't finish it. I forgot that the PR is remained open.)

Reply:
Fixed, thx.